src/main/java/supersymmetry/mixins/gregtech/MetaTileEntityLargeMinerMixin.java

```diff
@@ -0,0 +1,29 @@
+package supersymmetry.mixins.gregtech;
+
+import gregtech.api.metatileentity.multiblock.MultiblockWithDisplayBase;
+import gregtech.common.metatileentities.multi.electric.MetaTileEntityLargeMiner;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.asm.mixin.Mixin;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@Mixin(MetaTileEntityLargeMiner.class)
+public abstract class MetaTileEntityLargeMinerMixin extends MultiblockWithDisplayBase {
+
+    public MetaTileEntityLargeMinerMixin(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId);
+    }
+    public void addInformation(ItemStack stack, @Nullable World player, @NotNull List<String> tooltip, boolean advanced) {
```

Comment:
this should be changed to inject

Reply:
wait, i got this error:
*\Susy-Core\src\main\java\supersymmetry\mixins\gregtech\MetaTileEntityFluidDrillMixin.java:23: error: Unable to locate obfuscation mapping for @Inject target addInformation
    @Inject(method = "addInformation", at = @At("TAIL"))
    ^

Reply:
done